modules/migration-setting-up-target-cluster-to-accept-source-dns-domain.adoc

```diff
@@ -1,34 +1,35 @@
 // Module included in the following assemblies:
 //
 // * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-mtc.adoc
 
-[id="setting-up-target-cluster-to-accept-source-dns-domain_{context}"]
-== Setting up the target cluster to accept the source DNS domain
+[id="migration-setting-up-target-cluster-to-accept-source-dns-domain_{context}"]
+= Setting up the target cluster to accept the source DNS domain
 
-You can set up the target cluster to accept requests for migrated applications in the DNS domain of the source cluster.
+You can set up the target cluster to accept requests for a migrated application in the DNS domain of the source cluster.
 
 .Procedure
 
 For both non-secure HTTP access and secure HTTPS access, perform the following steps:
 
-. Create a route in the application’s project for the hostname that applications used in the source cluster:
+. Create a route in the target cluster's project that is configured to accept requests addressed to the application's FQDN in the source cluster:
 +
 [source,terminal]
 ----
-$ oc expose svc app1-svc --hostname app1.apps.ocp3.example.com \
- -n app1-namespace
+$ oc expose svc <app1-svc> --hostname <app1.apps.source.example.com> \
+ -n <app1-namespace>
 ----
 +
 With this new route in place, the server accepts any request for that FQDN and sends it to the corresponding application pods.
-In addition, when you migrate the application, another route is created in the target cluster domain (`app1.apps.ocp4.example.com`). Requests can reach the migrated application using both of these hostnames.
+In addition, when you migrate the application, another route is created in the target cluster domain (`app1.apps.target.example.com`). Requests reach the migrated application using either of these hostnames.
```

Comment:
sugg based on ISG:

> Avoid using parentheses in general text because text in parentheses is decreased in importance.

I think you can use "...cluster domain, such as `app1.apps.ocp4.example.com`."  My apologies for not noticing this one earlier.